src/components/WalletDropdown/DefaultMenu.tsx

```diff
@@ -0,0 +1,263 @@
+import { ButtonPrimary } from '../Button'
+import StatusIcon from '../Identicon/StatusIcon'
+import IconButton, { IconHoverText } from './IconButton'
+import { MenuState } from './index'
+
+const UNIbutton = styled(ButtonPrimary)`
+  background: linear-gradient(to right, #9139b0 0%, #4261d6 100%);
+  border-radius: 12px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-top: 12px;
+  color: white;
+`
+
+const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+`
+
+const ConnectButton = styled.button`
+  border: none;
+  outline: none;
+  border-radius: 12px;
+  height: 44px;
+  width: 288px;
+  background-color: ${({ theme }) => theme.accentAction};
+  color: white;
+  font-weight: 600;
+  font-size: 16px;
+  cursor: pointer;
+`
+
+const IconContainer = styled.div`
+  display: inline-block;
+  float: right;
+  & > a {
+    margin-right: 8px;
+  }
+  & > a:last-child {
+    margin-right: 0px;
+    ${IconHoverText}:last-child {
+      left: 0px;
+    }
+  }
+`
+
+const Divider = styled.div`
+  border: 0.5px solid ${({ theme }) => theme.backgroundOutline};
+  margin-top: 16px;
+  margin-bottom: 16px;
+`
+
+const ToggleMenuItem = styled.button`
+  background-color: transparent;
+  margin: 0;
+  border: none;
+  display: flex;
+  flex: 1;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 0px;
+  justify-content: space-between;
+  font-size: 14px;
+  font-weight: 400;
+  width: 100%;
+  margin-bottom: 8px;
+  color: ${({ theme }) => theme.textSecondary};
+  :hover {
+    cursor: pointer;
+    text-decoration: none;
+  }
+`
+
+const USDText = styled.div`
+  font-size: 16px;
+  font-weight: 500;
+  color: ${({ theme }) => theme.textSecondary};
+  margin-top: 8px;
+`
+
+const AuthenticatedHeader = () => {
+  const { account, chainId, connector } = useWeb3React()
+  const [isCopied, setCopied] = useCopyClipboard()
+  const copy = useCallback(() => {
+    setCopied(account || '')
+  }, [account, setCopied])
+  const dispatch = useAppDispatch()
+  const balanceString = useCurrencyBalanceString(account ?? '')
+  const {
+    nativeCurrency: { symbol: nativeCurrencySymbol },
+    explorer,
+  } = getChainInfoOrDefault(chainId ? chainId : SupportedChainId.MAINNET)
+
+  const { address: parsedAddress } = useENS(account)
+  const unclaimedAmount: CurrencyAmount<Token> | undefined = useUserUnclaimedAmount(parsedAddress)
+  const connectionType = getConnection(connector).type
+  const nativeCurrency = useNativeCurrency()
+  const nativeCurrencyPrice = useStablecoinPrice(nativeCurrency ?? undefined) || 0
+  const openClaimModal = useToggleModal(ApplicationModal.ADDRESS_CLAIM)
+
+  const amountUSD = useMemo(() => {
+    const price = parseFloat(nativeCurrencyPrice.toFixed(5))
+    const balance = parseFloat(balanceString || '0')
+    return price * balance
+  }, [balanceString, nativeCurrencyPrice])
+
+  return (
+    <>
+      <div style={{ marginBottom: 12 }}>
```

Comment:
can't tell if this was just copied from existing stuff, but otherwise should use styled-components for these inline styles. might be able to consolidate the parent divs a bit but i didn't look closely